cri/v1alpha2/cri.go

```diff
@@ -286,7 +288,10 @@ func (c *CriManager) RunPodSandbox(ctx context.Context, r *runtime.RunPodSandbox
 	}
 
 	// once sandbox container created, we are obligated to store it.
```

Comment:
this comment is for the `c.SandboxStore.Put(sandboxMeta);`?

Reply:
delete it for not necessary.  ^_^